### ISSUE TITLE
object not reference at OnModelCreating Error

### ISSUE THREAD

**Issue by mahyarmot** [Eastrall/EntityFrameworkCore.DataEncryption#10]
I have followed the package installation, however, I am receiving "Object reference not set to an instance of an object" at OnModelCreating event. The version of EF Core is 3.1.1 and the inner exception is 

**ChangeTracker = 'this.ChangeTracker' threw an exception of type 'System.InvalidOperationException'**

    ```
    private readonly byte[] _encryptionKey = null;
        private readonly byte[] _encryptionIV = null;
        private readonly IEncryptionProvider _provider;

        public ApplicationDbContext(DbContextOptions options)
        : base(options)
        {
            this._encryptionKey = Encoding.UTF8.GetBytes("....");
            this._encryptionIV = Encoding.UTF8.GetBytes("....");
            this._provider = new AesProvider(this._encryptionKey, this._encryptionIV);
        }

        protected override void OnModelCreating(ModelBuilder modelBuilder)
        {
            modelBuilder.UseEncryption(this._provider);
        }
```

**Comment by mahyarmot** [comment]
Having reviewed regarding an open bug, I tried to downgrade EF core version to version 3.0.0.0 but that did not solve the issue too.

**Comment by ekastimo** [comment]
After doing some troubleshooting,I think there is a bug in the extension method,  it appears that in the process of reading the properties from entities, some properties which are not discriminators also have PropertyInfo as null, in my case, it was owned classes. ( They have some sort of ID).

![image](https://user-images.githubusercontent.com/12415442/92593905-9a946500-f2aa-11ea-9c4d-60768da1fc43.png)

**Comment by Eastrall** [comment]
Thanks for the analysis. I'll add a verification on the `PropertyInfo` field. But if you want, you can open a PR to add an extra verification on the `PropertyInfo`.

**Comment by cesarsouza** [comment]
I understand a PR for this issue has been merged, does it mean it has been solved?

**Comment by Eastrall** [comment]
Yes indeed. It has been shipped with `v2.0.0`. Thanks for updating this. Closing the issue.
